### ISSUE TITLE
An issue with implementing Push Notification for ios.

### ISSUE THREAD

**Issue by navratanp7737** [RocketChat/Rocket.Chat.ReactNative#411]
Hi ,

I am trying to implement Push Notification with the ios app, as I tested the .pem file its ok and I am able to send push notification but as I am unable to get the push notification from rocket chat app. In fact, I don’t find any guide for how to activate push notification with rocket chat.

Here are the steps i followed.

Here are my rocket chat push settings :
![15 towerchat 1](https://user-images.githubusercontent.com/40238993/43892510-c6a04dea-9be9-11e8-9949-047aa87bed26.png)

![15 towerchat 2](https://user-images.githubusercontent.com/40238993/43892551-e1ba853c-9be9-11e8-8482-194f43bbd2b1.png)

by following these instructions.

https://github.com/raix/push/blob/v3.x/docs/IOS.md

Let me know where i wrong

**Comment by saadqbal** [comment]
Try Setting the **Enable Gateway** to false.

**Comment by q210** [comment]
Disabling gateway helped for iOS push notifications in our case too - we've deployed local rocket.chat server and built RN client for it - up until we needed to add Google FCM push notifications capability.

Sadly, Rocket.Chat server only supports deprecated Google GCM push notifications right now, and I've not found any way for FCM keys to work when setting them from Admin Panel.
In the end, we've deployed custom rocket chat gateway from this project https://github.com/arminfelder/RocketChatMobilePushGateway

**Comment by diegolmello** [comment]
@q210 We support FCM already. Can you try again? I'll close this one.

**Comment by rajkumarbhaji** [comment]
@diegolmello We have configured Firebase Server keys in RocketChat admin. And we downloaded Rocketchat REactNative code and built it. Now the background messages are working in Android, but on iOS it is not working. Do we have to follow any other stops.
Note: As per the documents, we have configured all the APNs settings, etc... for the preliminary test, we are able to push the messages from the Firebase console. But when we send messages from Rocket chat web to iOS device, then we are not receiving those messages when the app is in background mode. 

Kindly help us.
